independent-publisher-connectors/ORB Intelligence/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4731 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ORB Database",
+    "version": "v3",
+    "description": "Orb provides with a framework of API calls to use the Orb database for various use cases. The framework comprises five calls, Match, Fetch, Search, Lookalike and Corporate Tree.",
+    "contact": {
+      "name": "Aaryan Arora, Ankita Singh",
+      "email": "aaryan2134@gmail.com"
+    }
+  },
+  "host": "api.orb-intelligence.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/3/match/": {
+      "get": {
+        "parameters": [
+          {
+            "in": "query",
+            "name": "api_key",
+            "required": false,
+            "type": "string",
+            "default": "c66c5dad-395c-4ec6-afdf-7b78eb94166a",
+            "description": "API Access Key.",
+	          "x-ms-summary": "API Key"

```

Comment:
@aaryan2134 Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
Sorry but I am not able to clearly understand this. I just need to correct similar examples, if any, right?

Reply:
@mamurshe Thank you so much sir! That was a great help!
Made the changes

Reply:
@aaryan2134 @AnkitaSinghIE I can not see the changes for indentation issues. Please revisit and push the changes to PR.

Reply:
@sriyen-msft I think the changes were not visible due to the security validation queue not clearing up. Could you please check once again? 
I have updated the indentation using Prettify JSON plugin in VS Code for both the files.

Reply:
@aaryan2134 This is still not fixed - Please refer to [this](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) swagger for formation and alignment corrections.